schema/crdb/drop-service-table/up7.sql

```diff
@@ -0,0 +1,3 @@
+SET
+  LOCAL disallow_full_table_scans = OFF;
+UPDATE omicron.public.sled_resource SET kind = 'instance';
```

Comment:
Is this worse than having up6.sql add a `NOT NULL` and `DEFAULT` value and then removing the default here?  I would expect the approach here rewrites the `sled_resource` table, whereas setting a default and then removing it wouldn't.

Reply:
... yes, yes it is. Changed in [88aaf03](https://github.com/oxidecomputer/omicron/pull/5287/commits/88aaf03571656ed0020322a4bc7970a9a4f91d25)